.mailmap

```diff
@@ -2,6 +2,7 @@ Alexei Starovoitov <alexei.starovoitov@gmail.com>
 André Martins <andre@cilium.io>
 Andrew Sy Kim <kim.andrewsy@gmail.com>
 Arika Chen <eaglesora@gmail.com>
+Arthur Chiao <arthurchiao@hotmail.com>
 Arthur Evstifeev <aevstifeev@gitlab.com>
 Arthur Evstifeev <mail@ap4y.me>
```

Comment:
I'm not sure what this file is, but I noticed this person has two entries, while for example, Sergey has one entry with his emails consolidated. Feel free to ignore if this is not a thing to fix.

Reply:
Yeah I noticed this as well. Playing around with it, it seemed to make a slight difference but this might be due to another minor bug. I'll look at making this more consistent on the main branch.

Reply:
eh, if I'll respin for the below then I may as well fix this too.

Reply:
actually, it turns out the two different forms solve different problems and if I make them consistent, then the emails won't show up in authors correctly.